source/client-side-encryption/client-side-encryption.rst

```diff
@@ -231,11 +231,12 @@ csfle_ library is used. It speaks the MongoDB wire protocol and the driver uses
 mongocryptd_ by connecting with a MongoClient. By default, if csfle_ is
 unavailable, the driver should attempt to automatically spawn mongocryptd_. If
 the MongoClient is configured with `extraOptions.mongocryptdBypassSpawn` set to
-|true|, OR `bypassAutoEncryption` is set to |true| then the driver will not
-attempt to spawn mongocryptd_. The mongocryptd_ process is responsible for self
-terminating after idling for a time period. If |opt-csfle-required| is set to
-|true|, the driver will not connect to mongocryptd_ and instead rely on csfle_
-being available.
+|true|, OR `bypassAutoEncryption` is set to |true|, OR `bypassQueryAnalysis` is
+set to |true| then the driver will not attempt to spawn mongocryptd_.
+
+The mongocryptd_ process is responsible for self terminating after idling for a
+time period. If |opt-csfle-required| is set to |true|, the driver will not
+connect to mongocryptd_ and instead rely on csfle_ being available.
```

Comment:
Great catch. I suggest adding the following case to the "Bypass spawning mongocryptd" prose test.

```diff
diff --git a/source/client-side-encryption/tests/README.rst b/source/client-side-encryption/tests/README.rst
index cf123a0d..3b7c73d5 100644
--- a/source/client-side-encryption/tests/README.rst
+++ b/source/client-side-encryption/tests/README.rst
@@ -1007,6 +1007,11 @@ The following tests that setting ``bypassAutoEncryption=true`` really does bypas
 
 #. Validate that mongocryptd was not spawned. Create a MongoClient to localhost:27021 (or whatever was passed via ``--port``) with serverSelectionTimeoutMS=1000. Run a handshake command and ensure it fails with a server selection timeout.
 
+Via bypassQueryAnalysis
+```````````````````````
+
+Repeat the steps from the "Via bypassAutoEncryption" test, replacing "bypassAutoEncryption=true" with "bypassQueryAnalysis=true".
+
 Deadlock tests
 ~~~~~~~~~~~~~~
 
```
The Go implementation has been updated with proposed test https://github.com/kevinAlbs/mongo-go-driver/pull/2/commits/8750b31c6d6eac1785093b89698495e4115cbcfc

Reply:
Sure, done! :+1: